x-pack/platform/plugins/shared/cases/server/routes/api/user_actions/get_all_user_actions.ts

```diff
@@ -1,72 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import { schema } from '@kbn/config-schema';
-
-import type { DocLinksServiceSetup } from '@kbn/core/server';
-import type { userActionApiV1 } from '../../../../common/types/api';
-import { CASE_USER_ACTIONS_URL } from '../../../../common/constants';
-import { createCaseError } from '../../../common/error';
-import { createCasesRoute } from '../create_cases_route';
-import { DEFAULT_CASES_ROUTE_SECURITY } from '../constants';
-
-/**
- * @deprecated since version 8.1.0
- */
-export const getUserActionsRoute = ({
-  isServerless,
-  docLinks,
-}: {
-  isServerless?: boolean;
-  docLinks: DocLinksServiceSetup;
-}) =>
-  createCasesRoute({
-    method: 'get',
-    path: CASE_USER_ACTIONS_URL,
-    security: DEFAULT_CASES_ROUTE_SECURITY,
-    params: {
-      params: schema.object({
-        case_id: schema.string(),
-      }),
-    },
-    options: { deprecated: true },
-    routerOptions: {
-      access: isServerless ? 'internal' : 'public',
-      summary: 'Get case activity',
-      description: `Returns all user activity for a case.`,
-      // You must have `read` privileges for the **Cases** feature in the **Management**, **Observability**, or **Security** section of the Kibana feature privileges, depending on the owner of the case you're seeking.
-      tags: ['oas-tag:cases'],
-      deprecated: {
-        documentationUrl: docLinks.links.cases.legacyApiDeprecations,
-        severity: 'warning',
-        reason: {
-          type: 'migrate',
-          newApiMethod: 'GET',
-          newApiPath: '/api/cases/<case_id>/user_actions/_find',
-        },
-      },
-    },
-    handler: async ({ context, request, response }) => {
-      try {
-        const caseContext = await context.cases;
-        const casesClient = await caseContext.getCasesClient();
-        const caseId = request.params.case_id;
-
-        const res: userActionApiV1.CaseUserActionsDeprecatedResponse =
-          await casesClient.userActions.getAll({ caseId });
```

Comment:
Should `getAll` be removed from `x-pack/platform/plugins/shared/cases/server/client/user_actions/client.ts`? I'm not sure it is used anymore.

Reply:
It is still used in `x-pack/platform/plugins/shared/cases/server/client/cases/push.ts` and `x-pack/platform/plugins/shared/cases/server/client/user_actions/get.ts`

Reply:
Ah, all good then 🙌 I thought it had been replaced by `find`